### PR TITLE
During requestAnimationFrame detect if session has ended

### PR DESCRIPTION
**Pull request by cabanier**
***
<a href="https://pr-preview.s3.amazonaws.com/cabanier/webxr-1/pull/1169.html" title="Last updated on Feb 3, 2021, 7:30 PM UTC (426fb22)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/webxr/1169/ffe5171...cabanier:426fb22.html" title="Last updated on Feb 3, 2021, 7:30 PM UTC (426fb22)">Diff</a>

### CODE REVIEW

**Comment by cabanier** [comment]
@toji Closes #1168

File: index.html [hunk 1]
```diff
@@ -0,0 +1,8811 @@
+<!doctype html><html lang="en">
```
Comment (Manishearth) [inline index.html:1]: don't check in index.html

**Review by Manishearth — APPROVED** [review]
r=me but remove index.html

**Comment by cabanier** [comment]
@Manishearth I removed index.html

**Review by toji — COMMENTED** [review]
Post-merge LGTM, for what it's worth. Thanks!

**Pull request merged**